docking/dock_setup.py

```diff
@@ -47,15 +45,14 @@ def combine_potentials(
     """
 
     host_potentials, host_masses = openmm_deserializer.deserialize_system(
-        host_system,
-        precision,
-        cutoff=1.0
+        host_system, precision, cutoff=1.0
     )
+    for atom_num in constant_atoms:
```

Comment:
Two notes:

1. I'm going to eventually refactor this function soonish so there will be a single combine_potentials function used across multiple workflows (abfe/rbfe) - so it may make sense to do the constant atoms juggling outside of this function.

2. Are idxs in constant_atoms 1-idxed for the protein atoms?

Reply:
1. moved this back to pose_dock.py
2. Yes, because PDB files start at ATOM 1